packages/components/src/components/form/InputStyles.ts

```diff
@@ -0,0 +1,91 @@
+import { darken } from 'polished';
+import styled, { css } from 'styled-components';
+import { FONT_FAMILY, FONT_WEIGHT, NEUE_FONT_SIZE } from '../../config/variables';
+import { InputState, InputVariant } from '../../support/types';
+import { getInputStateColor } from '../../utils';
+import { getInputStateBGColor, getInputStateBorderColor } from '../../utils/colors';
```

Comment:
do we need this functions hidden somewhere in utils when they are closely tied to this component?

Reply:
Actually not, you are right.

Reply:
done